freeglut/freeglut/src/mswin/fg_main_mswin.c

```diff
@@ -1375,10 +1458,13 @@ LRESULT CALLBACK fgPlatformWindowProc( HWND hWnd, UINT uMsg, WPARAM wParam, LPAR
             break;
 
         fgState.Modifiers = fgPlatformGetModifiers( );
-        INVOKE_WCB( *window, Keyboard,
-                    ( (char)wParam,
-                      window->State.MouseX, window->State.MouseY )
+        INVOKE_WCB( *window, KeyboardExt,
+                ( wParam, window->State.MouseX, window->State.MouseY )
         );
+        if (wParam < 128)
```

Comment:
Same as other comment:
Will this break existing GLUT functionality? I don't think I've tried Unicode keys so I don't know how this shows up currently.